### PR TITLE
11 get articles topic query

### PR DESCRIPTION
**Pull request by addleton**

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -320,3 +320,33 @@ describe("GET /api/users", () => {
...
+    });
+    test("404: responds with a message when passed a query topic that does not exist", () => {
+        return request(app)
+            .get("/api/articles?topic=pepsi")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Not found");
+            });
+    });
+});
```
Comment (Harrison365) [inline __tests__/integration.test.js:352]: Nice :) 
We will also need a successful 200 test for a topic which has nor articles but still exists. An empty array should be returned.

**Comment by Harrison365** [comment]
Hi Michael. Just the one extra test needed. Please send us another PR when you'd like us to check this as its an important one 😄 thanks 👍

File: __tests__/integration.test.js (hunk 1 above)
Comment (addleton) [reply __tests__/integration.test.js:352]: I'm on it :)

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -320,3 +320,41 @@ describe("GET /api/users", () => {
...
+    });
+    test("200: responds with an empty array when no articles contain the topic passed in", () => {
+        return request(app)
+            .get("/api/articles?topic=paper")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles).toEqual([]);
+            });
+    });
+    test("400: responds with a message when passed an invalid column in query", () => {
```
Comment (Hard-Luck) [inline __tests__/integration.test.js]: this test isn't a test for topic query as it's not using the topic=, it's nice to see you thinking about it but I would delete this test

**Review by Hard-Luck — APPROVED** [review]
great changes, read my comment, then once done you can merge :)

File: __tests__/integration.test.js (hunk 2 above)
Comment (addleton) [reply __tests__/integration.test.js]: Done :)

**Pull request merged**